tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -1422,8 +1452,21 @@ void SetBranchesHelper(TTree *inputTree, TTree &outputTree, const std::string &i
       // must construct the leaflist for the output branch and create the branch in the output tree
       auto *const leaf = static_cast<TLeaf *>(inputBranch->GetListOfLeaves()->UncheckedAt(0));
       const auto bname = leaf->GetName();
-      const auto counterStr =
-         leaf->GetLeafCount() ? std::string(leaf->GetLeafCount()->GetName()) : std::to_string(leaf->GetLenStatic());
+      auto *sizeLeaf = leaf->GetLeafCount();
+      const auto sizeLeafName = sizeLeaf ? std::string(sizeLeaf->GetName()) : std::to_string(leaf->GetLenStatic());
```

Comment:
FYI, for a leaf `[4][4]`, `leaf->GetLenStatic()` returns 16. I guess that's good?

Reply:
I...don't know, we don't have test coverage for 2-dimensional arrays in RDF